src/Umbraco.Web.UI.Client/src/views/components/umb-grid-selector.html

```diff
@@ -24,15 +24,15 @@
 
         <a href="" class="umb-grid-selector__item -placeholder" ng-if="(availableItems | compareArrays:selectedItems:'alias').length > 0" ng-click="openItemPicker($event)" hotkey="alt+shift+g">
             <div class="umb-grid-selector__item-content">
-                <div class="umb-grid-selector__item-label -blue" ng-if="defaultItem !== null"><localize key="grid_chooseExtra">Choose extra</localize> {{ itemLabel }}</div>
-                <div class="umb-grid-selector__item-label -blue" ng-if="defaultItem === null"><localize key="grid_chooseDefault">Choose default</localize> {{ itemLabel }}</div>
+                <div class="umb-grid-selector__item-label -blue" ng-if="defaultItem !== null"><localize key="grid_chooseExtra">Choose extra</localize>&nbsp;{{ itemLabel }}</div>
+                <div class="umb-grid-selector__item-label -blue" ng-if="defaultItem === null"><localize key="grid_chooseDefault">Choose default</localize>&nbsp;{{ itemLabel }}</div>
             </div>
         </a>
 
     </div>
 
     <div class="text-center" ng-if="(availableItems | compareArrays:selectedItems:'alias').length === 0">
-        <small><localize key="general_all">Akk</localize> {{itemLabel}}s <localize key="grid_areAdded">are added</localize></small>
+        <small><localize key="general_all">Akk</localize>&nbsp;{{itemLabel}}s <localize key="grid_areAdded">are added</localize></small>
```

Comment:
The default text for `general_all` also isn't English:
```suggestion
        <small><localize key="general_all">All</localize>&nbsp;{{itemLabel}}s <localize key="grid_areAdded">are added</localize></small>
```

Reply:
I did not realise the Akk is a typo. I will update that as well